plugins/postcss-is-pseudo/INSTALL.md

```diff
@@ -0,0 +1,164 @@
+# Installing PostCSS Is Pseudo
+
+[PostCSS Is Pseudo] runs in all Node environments, with special
+instructions for:
+
+| [Node](#node) | [PostCSS CLI](#postcss-cli) | [Webpack](#webpack) | [Create React App](#create-react-app) | [Gulp](#gulp) | [Grunt](#grunt) |
+| --- | --- | --- | --- | --- | --- |
+
+## Node
+
+Add [PostCSS Is Pseudo] to your project:
+
+```bash
+npm install postcss @csstools/postcss-is-pseudo --save-dev
```

Comment:
Any strong opinions on adding the org vs not adding the org?

Reply:
I like the org for multiple reasons :

- [avoiding substitution attacks](https://github.blog/2021-02-12-avoiding-npm-substitution-attacks/) (we would not be the victim but without scoping package names we can contribute to the issue)
- name recognition.
- we don't have to pick a less logical package name if the best option is already taken without scope.